utils/common.go

```diff
@@ -32,6 +40,8 @@ var (
 	uuidSeed    [24]byte
 	uuidCounter uint64
 	uuidSetup   sync.Once
+	unitsMap    = map[string]int64{"k": kb, "m": mb, "g": gb, "t": tb, "p": pb}
+	sizeRegex   = regexp.MustCompile(`(?i)^(\d+(\.\d+)*) ?([kmgtp])?b?$`)
```

Comment:
Do we really need to use regex here?
Can you explain more about this regex? I do like to make many tests using strings instead regex.

Reply:
(based on my strong experience in regexp patterns)

This regexp is simple pattern for extracting float value and unit name

So, something like:

(strict begin)(int or float value)(maybe space symbol)(maybe unit letter)(maybe b symbol)(string end)

So, any next examples are correct:
```
42
42m
42 m
42 Mb
42.5 Mb
```

Pattern is correct and many developers use it in different packages, frameworks and languages.

And, I really don't know how parse human readable value using strings.

Reply:
i tried the string recognition
https://github.com/gofiber/fiber/pull/1875/commits/4491602d7ef78869e269282705084e22680f3be9

```go
old:
Benchmark_ConvertToBytes/fiber-8                 3541161                317.9 ns/op          128 B/op          2 allocs/op
Benchmark_ConvertToBytes/fiber-8                 3465039             336.8 ns/op          128 B/op          2 allocs/op

new:
Benchmark_ConvertToBytes/fiber-12               32883782            33.76 ns/op              0 B/op          0 allocs/op
Benchmark_ConvertToBytes/fiber-12               36084900           33.47 ns/op              0 B/op          0 allocs/op
```

Reply:
@webdevium are you okay with the new code ? is now a little more, but also 10 times faster

Reply:
@ReneWerner87 Awesome!
I don't have enough golang experience to do things like this without regular expressions :) 

Thank you for your help.

Reply:
@ReneWerner87 But method has panic if human string is empty :(